<launch>
   <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
      <param name="port" value="/dev/ttyUSB0"/>
      <param name="baud" value="57600"/>
   </node>

   <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" />

   <node name="wheel_control_output" pkg="hektar" type="wheel_control.py" output="screen" > 
     <param name="speed" value="70" />
   </node>

   <node name="ir_error_node" pkg="hektar" type="ir_error.py" output="screen" />

   <!-- <node name="ir_fork_analysis" pkg="hektar" type="fork_analysis.py" output="screen" /> -->

   <!-- <node name="control_master" pkg="hektar" type="master_control.py" output="screen" /> -->
</launch>
